Forge.TreeWalker.UnitTests/test/ActionsAndCallbacks/ForgeUserContext.cs

```diff
@@ -48,7 +48,7 @@ public Task<int> GetCountAsync()
             return Task.FromResult(2);
         }
 
-        public IDictionary<string, string> GetDictionary()
+        public Dictionary<string, string> GetDictionary()
```

Comment:
>Dictionary [](start = 15, length = 10)

just curious, why do you change from "IDictionary" ?

Reply:
Oh oops. I was testing something else before I started the csproj changes. I'll remove this.

---
In reply to: [584957321](https://github.com/microsoft/Forge/pull/40#discussion_r584957321) [](ancestors = 584957321)